Sources/VaporSecurityHeaders/Configurations/HSTSRedirectMiddleware.swift

```diff
@@ -0,0 +1,23 @@
+import Vapor
+
+public class HSTSRedirectMiddleware: Middleware {
```

Comment:
I think I'm happy with this being a separate middleware, but it should probably be renamed `HTTPSRedirectMiddleware` since it doesn't actually do anything with HSTS even though it's related